Neuron/zh-Hans.lproj/Localizable.strings

```diff
@@ -44,3 +44,19 @@
 "CustomTokenError.badNameError" = "Token名称不正确";
 "CustomTokenError.badSymbolError" = "Token symbol不正确";
 "CustomTokenError.undefinedError" = "未知错误";
+
+// Wallet
```

Comment:
Add these to `en.lproj` too (can be exact the same Chinese copy for later transaction).